firefox-ios/firefox-ios-tests/Tests/XCUITests/CreditCardsTests.swift

```diff
@@ -370,11 +373,95 @@ class CreditCardsTests: BaseTestCase {
         }
     }
 
-    private func fillCardDetailsOnWebsite(cardNr: String, expirationDate: String, nameOnCard: String) {
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2306980
+    func testUpdatePrompt() {
```

Comment:
The test is a bit long, not sure if it can be splitted somehow, if not let's keep an eye on it in case it starts being flaky

Reply:
if it was a smoke test i would have considered to split it, but being functional only i would say to leave it as it is and monitor it, but i did run it 50 times in a row and i have 100% pass rate.

Reply:
sounds good to me